climada/hazard/tc_tracks.py

```diff
@@ -758,13 +758,17 @@ def read_simulations_emanuel(self, *args, **kwargs):
         self.__dict__ = TCTracks.from_simulations_emanuel(*args, **kwargs).__dict__
 
     @classmethod
-    def from_simulations_emanuel(cls, file_names, hemisphere=None):
+    def from_simulations_emanuel(cls, file_names, subset=None, hemisphere=None):
```

Comment:
Although optional arguments _should always_ be called by name, and this would teach everybody a lesson who doesn't follow this maxim - moving `subset` to the end of the argument list would prevent regression (in arguably very rare cases).

Reply:
Thanks, agreed and done!